libs/wasm-detective/src/matcher_pii_payments.rs

```diff
@@ -97,10 +102,157 @@ pub fn usa_bank_routing_number(_request: &Request, field: Value) -> Result<bool,
     Ok(cleaned_string.len() == 9)
 }
 
-pub fn crypto_currency_address(_request: &Request, _field: Value) -> Result<bool, CustomError> {
-    // TODO: can't find a reliable lib for handling these
-    Err(CustomError::Error("Not implemented".to_string()))
+pub fn crypto_currency_address(_request: &Request, field: Value) -> Result<bool, CustomError> {
+    // This function is only intended to capture the main blockchains
+    // We should eventually abstract
+    let address = field.str().trim();
+
+    // Ethereum
+    if address.starts_with("0x") {
+        return crypto_eth(address);
+    }
+
+    // Bitcoin
+    if address.starts_with("bc1") {
+        return _crypto_btc_bech32(address);
+    }
+    if address.starts_with('1') || address.starts_with('3') {
```

Comment:
This seems like it'll hit a lot of false positives or is this for specific fields?